proxywasm/v2/imports.go

```diff
@@ -70,15 +75,15 @@ func RegisterImports(instance common.WasmInstance) {
 	_ = instance.RegisterFunc("env", "proxy_call_custom_function", ProxyCallCustomFunction)
 }
 
-func ProxyLog(instance common.WasmInstance, logLevel LogLevel, messageData int32, messageSize int32) Result {
+func ProxyLog(instance common.WasmInstance, logLevel int32, messageData int32, messageSize int32) Result {
```

Comment:
is there any reason for downgrading LogLevel to int32?

Reply:
Wasm instance uses int32 param. If not downgrading , it will throw
```
[ERROR] [wasmer][instance] RegisterFunc recover func call: proxy_log, r: reflect: Call using int32 as type v2.LogLevel, stack: goroutine ...
```